lib/util/i18n.js

```diff
@@ -34,13 +30,25 @@ function getMatchingLocaleString (locale = '', defaultLocale = 'en-US') {
  *   id => string.
  * FIXME: Load languages on demand using fetch.
  */
-export async function loadLocaleData (locale, customMessages) {
-  const matchedLocale = getMatchingLocaleString(locale)
+export async function loadLocaleData(locale, customMessages, configLocales) {
+  const matchedLocale = getMatchingLocaleString(locale, 'en-US', configLocales)
   let messages
   switch (matchedLocale) {
+    case 'es-ES':
```

Comment:
Since we are adding string files for other non-[US English] languages, probably we should remove the country-specific code. `fr-FR` designates French text with France-specific messages (e.g. acronyms or terms only used in the country), but since we are providing examples used in the generic context, the locale and the message file name should just be `fr` going forward. Same for the other languages, and same in `config.yml`.

Reply:
Removed the country codes for most in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/2ba2f5eeb9da864347789e418be2bc16ab16ce6b. Kept the codes for en-US bc some spellings and terms are US specific, and I figured your french was likewise FR specific, @binh-dam-ibigroup